src/main/java/by/exadel/internship/service/InternshipService.java

```diff
@@ -0,0 +1,41 @@
+package by.exadel.internship.service;
+
+
+import by.exadel.internship.dto.internshipDTO.GuestInternshipDTO;
+import by.exadel.internship.entity.Internship;
+import by.exadel.internship.mapper.InternShipMapper;
+import by.exadel.internship.repository.InternshipRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class InternshipService {
+
+    private final InternshipRepository internshipRepository;
+    private final InternShipMapper internShipMapper;
+
+    @Autowired
+    public InternshipService(InternshipRepository internshipRepository, InternShipMapper internShipMapper) {
```

Comment:
the same with lombok

Reply:
done
![image](https://user-images.githubusercontent.com/65280608/113557877-62b68b80-9607-11eb-92bd-4341d11c0a41.png)